src/lib/main/main-ready-for-review.js

```diff
@@ -0,0 +1,46 @@
+const { askToReview } = require("../review");
+const { addLabels } = require("../label");
+const { isCIRequired } = require("../ci");
+const { isFirstPR } = require("../pr");
+
+async function pullRequestReopened(context) {
+  const comments = await context.config("bot-files/comments.yml");
+  context.github.issues.createComment(context.issue({ body: comments.prEdit }));
+  commonFunc(context);
+}
+
+async function pullRequestOpened(context) {
+  const comments = await context.config("bot-files/comments.yml");
+  if (await isFirstPR(context)) {
+    context.github.issues.createComment(
+      context.issue({ body: comments.prFirstTimeContributor })
+    );
+  }
+  commonFunc(context);
+}
+
+async function pullRequestChanged(context) {
+  const comments = await context.config("bot-files/comments.yml");
+  context.github.issues.createComment(context.issue({ body: comments.prEdit }));
+  commonFunc(context);
+}
+async function commonFunc(context) {
+  const comments = await context.config("bot-files/comments.yml");
```

Comment:
I would add to commonFunc
```
if(shouldComment) {
  const comments = await context.config(file);
  context.github.issues.createComment(context.issue({ body: comments[commentField] }));
}
```
add three new arguments to `commonFunc`, `file` `commentField` and `shouldComment`
and I would refactor `pullRequestReopened` to
```
async function pullRequestReopened(context) {
  commonFunc(context, "bot-files/comments.yml", 'prEdit', true);
}
```
and `pullRequestOpened` to
```
async function pullRequestOpened(context) {
  commonFunc(context, "bot-files/comments.yml", 'prFirstTimeContributor', await isFirstPR(context));
}
```